src/laboratory/models.py

```diff
@@ -16,6 +18,30 @@
 from presentation.models import AbstractOrganizationRef
 from . import catalog
 
+def upload_security_sheets(instance, filename):
```

Comment:
Too many functions with code too similar, maybe you need to refactor part of this functions and call here.